src/PowerShell/Microsoft.WinGet.Client/Microsoft.WinGet.Client.csproj

```diff
@@ -67,5 +69,13 @@
     <!-- Ensure Support for Windows 10, Version 1809 -->
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
 	</PropertyGroup>
+
+	<Target Name="CopyWinRTAct" AfterTargets="Build" Condition="'$(TargetFramework)' != ''">
+		<ItemGroup>
+			<WinRTActDll Include="$(SolutionDir)_build\$(Platform)\Debug\winrtact.dll" />

```

Comment:
Did you also add a build dependency on that project at some point?

Reply:
Yes, `winrtact.dll` comes from the `UndockedRegFreeWinRT` project (added as a `ProjectReference`) and across `Debug` and `Release`, it should only build in `Debug`. That should be set in the configuration manager. The `winrtact.dll` will end up being somewhere in the source directory with `$(Platform)` in its path, so I've just added an event to copy `**\$(Platform)\**\winrtact.dll` to the output directory (`net5.0-windows10.0.22000.0` and `net461`). It will probably find multiple `winrtact.dll` files. But since they should all be built for `$(Platform)` in `Debug`, they will just overwrite each other & it should be fine.